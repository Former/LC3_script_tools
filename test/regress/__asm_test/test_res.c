void func(void)
{
    printf("Test RES\n");
    __asm {
        AND R0, R0, #0 
        ADD R0, R0, #10 
        RES R0, #20
    }
}

int main()
{
    func();
    return 0;
}
