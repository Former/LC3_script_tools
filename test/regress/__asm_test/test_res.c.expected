Test RES
ReservedOperation reg = 10 val = 20
